Cargo.toml

```
@@ -38,9 +38,9 @@ id-arena = "2.2"
 itertools = "0.11"
 lazy_static = "1.4"
 libc = "0.2.147"
-melior = "0.9.9"
-mlir-sys = "0.2.0"
-num-bigint = "0.4.3"
+melior = { version = "0.12.0", features = ["ods-dialects"] }
+mlir-sys = "0.2.1"
+num-bigint = "0.4"
```

Shouldn't it be better to leave the full version?
```suggestion
num-bigint = "0.4.4"
```